.travis.yml

```diff
@@ -7,11 +7,11 @@ addons:
   chrome: stable
 
 before_script:
-  - npm run bootstrap
+  - npm install
```

Comment:
I think we can just remove this line now. Travis will automatically run `npm ci` (since we have a `package-lock.json`, otherwise it would automatically run `npm install`), so we want to avoid installing twice (https://docs.travis-ci.com/user/languages/javascript-with-nodejs/). I confirmed that `postinstall` is run during `npm ci` also.

Reply:
Oh I don't know about this thank you :)
I reflected it!